docs/database-setup.md

```diff
@@ -1,51 +1,80 @@
 # Setting up a database
 
-PostgreSQL can be a bit of a pain to configure if you don't know where to begin.
-Basically, your goal is to create a database, create a user account that has
-access to that database, and allow access to that user account over a network
-connection (which means it must have a password).
+If you're reading this, you're taking on a role of DBA at this project.
+Congrats, now buckle up and put on your shit kickers.
 
-The following instructions have been tested to work on Ubuntu 16.04, but will
-probably work on other Debian-based distros too.
-
-## Installation
+## Local - Dev Only
+Okay, maybe you got scared of that DBA thing and you're just a dev.
+Do this:
 
     $ sudo apt-get install postgresql
-
-## Creating a user account
-
     $ sudo -u postgres createuser -P <username>
-
-You will be prompted for a password that you'll need later.
-
-## Creating a database
-
     $ sudo -u postgres createdb -O <username> <database name>
-
-This database will be owned by the user you've specified. We recommend the
-database name be the same as the username.
-
-## Testing access
-
-To test that you're able to connect (type `\q` to quit after you log in):
-
+    # test it's there
     $ psql -U <username> -h localhost <database name>
 
-You can omit the database name here if it's the same as the username.
-
-## Database URL
-
-Finally, once everything is figured out, the `DATABASE_URL` parameter you're
-prompted for during install will look something like this:
-
-    pgsql://<username>:<password>@localhost/<database name>?serverVersion=9.5
-
-The server version should be set to the version of PostgreSQL provided by your
-distro. For Ubuntu 16.04, it's '9.5'. If unsure, you can omit this parameter
-entirely and things should still work.
-
-## Other resources
-
-* <https://help.ubuntu.com/community/PostgreSQL>
-* <https://www.postgresql.org/docs/9.5/static/app-createdb.html>
-* <https://www.postgresql.org/docs/9.5/static/app-createuser.html>
+Modify your .env with following URL:
+
+    pgsql://<username>:<password>@localhost/<database name>?serverVersion=9.6
+
+Enjoy!
+
+## Real Deal
+
+Still here? Awesome. Get Some!
+
+     // Create Users
+     CREATE ROLE youwish_devuser WITH LOGIN PASSWORD 'oooh_heees_trying';
```

Comment:
Are we going to be able to have 2 users per level?  One user that can do the schema creation (run the Symfony migrations) and a different user that is the one that the application normally runs as?  The user created with this would be the "migrations" user - it's the database owner.  The other one could actually be created by the "migrations" user and it would be the one the app actually runs as (the "app" user).  I would not want the website/app running as this user on a general basis - because this user is able to drop the database.

Reply:
dev and test migrations==readWriteUser

for prod - there's _prodadmin and _prod